src/app/relogin-popup/relogin-popup.service.ts

```diff
@@ -0,0 +1,56 @@
+import { Injectable, ErrorHandler } from '@angular/core';
+import { HttpClient, HttpHeaders, HttpParams, HttpErrorResponse } from '@angular/common/http';
+import { Observable } from 'rxjs';
+
+import { Response } from '../../../api-objects/GenericResponse';
+import { catchError, map, tap } from 'rxjs/operators';
+import { Utilities } from '../common/utlilities';
+import { LoginSession } from '../../../api-objects/LoginSession';
+import { authorityTypes } from '../../../api-objects/UsersObject';
+import baseUrl from '../common/base-url';
+
+@Injectable()
+export class ReloginPopupService {
+
+    constructor(private http: HttpClient) { }
+
+    public login(username: string, password: string): Observable<null> {
```

Comment:
I think you should call use the login service already implemented to set up session storage.